sbbbattlesim/characters/__init__.py

```diff
@@ -117,6 +117,20 @@ def generate_attack(self, source, target, reason, attacking=False):
             damage=self.attack,
         )
 
+    def to_state(self, slot):
```

Comment:
I noticed this isn't needed since Player handles turning characters to json.

Reply:
this should now be addressed